test/MUXControlsTestApp/Utilities/EntityPropertiesControl/EntityPropertiesControl.xaml.cs

```diff
@@ -215,6 +215,7 @@ private void GenerateUI()
                     rowIndex++;
                 }
 
+#if !ARM64
```

Comment:
What is the reason for the #if checks in this file? Please add a comment.

Reply:
BindingFlags is not found. I'll add comments.

---
In reply to: [255264217](https://github.com/Microsoft/microsoft-ui-xaml/pull/288#discussion_r255264217) [](ancestors = 255264217)

Reply:
Why is it ok to exclude these chunks of these functions? It seems weird to introduce these behavioral differences between these functions on arm64. Would it make more sense to throw E_NOTIMPL instead of just silently skipping over the problematic code?

---
In reply to: [255264982](https://github.com/Microsoft/microsoft-ui-xaml/pull/288#discussion_r255264982) [](ancestors = 255264982,255264217)